test/trento_web/controllers/v1/users_controller_test.exs

```diff
@@ -11,11 +12,54 @@ defmodule TrentoWeb.V1.UserControllerTest do
   @endpoint TrentoWeb.Endpoint
 
   setup %{conn: conn} do
+    delete_default_abilities()
+
+    conn =
+      conn
+      |> Plug.Conn.put_private(:plug_session, %{})
+      |> Plug.Conn.put_private(:plug_session_fetch, :done)
+      |> Pow.Plug.put_config(otp_app: :trento)
+
     api_spec = ApiSpec.spec()
 
+    # Default inject all:all abilities user
+    %{id: user_id} = insert(:user)
+    %{id: ability_id} = insert(:ability, name: "all", resource: "all")
+    insert(:users_abilities, user_id: user_id, ability_id: ability_id)
+
+    conn =
+      Pow.Plug.assign_current_user(conn, %{"user_id" => user_id}, Pow.Plug.fetch_config(conn))
+
     {:ok, conn: put_req_header(conn, "accept", "application/json"), api_spec: api_spec}
   end
 
+  describe "forbidden response" do
```

Comment:
I have my shared feelings on this.
I think that the policy should be tested in isolation and pure in `test/policy_test` kind of file. Where you can test any possible combination.
But i understand that you want to test that the policy is applied properly.
I see that for example we don't test that the `users:all` doesn't allow the call, which for me is a bad thing.
Maybe we should test both?

Reply:
Yes, you're right, I will test both, full policy with unit tests, and policy controller integration with controller tests

Reply:
done